experimental/cloudstrapper/README.md

```diff
@@ -234,7 +234,7 @@
 
     agw-configure: configures the AGW to include controller information Command:
     ```
-    ansible-playbook agw-configure.yaml -i <DynamicInventoryFile> -e "agw=tag_Name_<GatewayId>" -e "dirLocalInventory=<WORK_DIR>" -u ubuntu
+    ansible-playbook agw-configure.yaml -i <DynamicInventoryFile> -e "agw=tag_Name_<GatewayId>" -e "dirLocalInventory=<WORK_DIR>" -u ubuntu [ -e KMSKeyID=<KEY_ID_TO_ADD_TO_SSH_AUTHORIZED> -e sshKey=<PATH-TO-PUBLIC-KEY-TO-ADD>]
```

Comment:
Perhaps adding a reference to tag platform if no keys need to be added?

Reply:
By default it will use the platform tag and not run the add keys if none are defined. I added a tag to use skip-tag in our devops usecase.

Reply:
Good point and I agree. Resolving this.